bootstrap/farmos.py

```
@@ -0,0 +1,165 @@
+def uv_sync_backend(skip_sync: bool) -> None:
+    if skip_sync:
+        info("uv sync 생략 (--skip-sync)")
+        return
+    info("FarmOS backend 의존성 동기화(uv sync)")
+    run_command(["uv", "sync"], cwd=BACKEND_DIR)
+
+
+def drop_farmos_tables(db_conf: dict[str, str]) -> None:
+    info("FarmOS 테이블 삭제(drop)")
+    # legacy: 기존 mock 농약 캐시 테이블도 함께 제거
+    targets = list(FARMOS_TABLES) + ["pesticide_products"]
```

Ruff (0.15.10)

[warning] Consider `[*list(FARMOS_TABLES), "pesticide_products"]` instead of concatenation

Replace with `[*list(FARMOS_TABLES), "pesticide_products"]`

(RUF005)